.circleci/config.yml

```diff
@@ -143,6 +188,46 @@ jobs:
             # Run the tests.
             bash -x ./nginx_integration_test.sh
 
+  test_coverage:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Build source dependencies
+          command: |
+            ./scripts/install_dependencies.sh
+      - run:
+          name: Build (with cmake) and test with coverage
+          command: |
+            rm -rf .build
+            scripts/run_coverage.sh
+            rm -rf /tmp/coverage
+            mkdir /tmp/coverage
+            tar czf /tmp/coverage/report.tar.gz coverage/report
+      - persist_to_workspace:
+          root: /tmp/
+          paths:
+            - coverage
+      - store_artifacts:
+          path: /tmp/coverage
+            
+  stat:
+      <<: *defaults
+      steps:
+        - run:
+            name: Print information about the execution environment
+            command: |
+              set -x
+              env | sort
+              nproc
+              getconf _NPROCESSORS_ONLN
+              cat /sys/fs/cgroup/cpuset/cpuset.cpus
+              cat /proc/self/cgroup
+              cat /proc/cgroups
+              find /sys/fs/cgroup
```

Comment:
Is this useful or just noise?

Reply:
Mostly noise.  This "stat" job was me probing the execution environment to try to figure out how to get the cgroup-imposed CPU count limit.  I didn't figure it out, but I decided to leave the job.  I don't see it being that valuable, but it's cheap enough to keep.  I'll edit it down for length, I think.